app/services/ember-cli.js

```diff
@@ -42,9 +42,17 @@ export default Em.Service.extend({
 
       if (errors.length) {return reject(errors);}
 
-      // Add app, config
+      // Add app
       out.push(this.compileJs(blueprints.app, 'demo-app/app'));
+
+      // Add router
+      if (!gist.get('files').findBy('nameWithModule', 'demo-app/router')) {
+        out.push(this.compileJs(blueprints.router, 'demo-app/router'));
+        console.error('Missing a router.js See issue https://github.com/ember-cli/ember-twiddle/issues/55#issuecomment-121724317');
```

Comment:
Actually, I have a better idea. If we added a blueprint, we could make router.js permanently optional.